boards/b-l072z-lrwan1/include/periph_conf.h

```diff
@@ -147,6 +147,27 @@ static const spi_conf_t spi_config[] = {
 #define SPI_NUMOF           ARRAY_SIZE(spi_config)
 /** @} */
 
+/**
+ * @name   ADC configuration
+ *
+ * Note that we do not configure all ADC channels,
+ * we just define 6 ADC channels, for the Nucleo
+ * Arduino header pins A0-A5
+ *
+ * @{
+ */
+static const adc_conf_t adc_config[] = {
+    {GPIO_PIN(PORT_A, 0), 0},
+    {GPIO_PIN(PORT_A, 1), 1},
+    {GPIO_PIN(PORT_A, 4), 4},
+    {GPIO_PIN(PORT_B, 0), 0},
+    {GPIO_PIN(PORT_C, 1), 1},
+    {GPIO_PIN(PORT_C, 0), 0},
+};
```

Comment:
I'm comparing the pin configured with UM2115. Are you sure this is working ? Especially A1, A3-A5 ?

![image](https://user-images.githubusercontent.com/1375137/106179377-3d3a7b00-619b-11eb-9eef-31533336c77c.png)

Reply:
Indeed in this board only A0 and A2 are accessible. Though, I made the change to match the table.